zulip/zulip/examples/delete-topic 

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import argparse
+import zulip
+
+
+usage = """delete-topic <stream_id/stream_name, topic_name>
+
+Example: delete-topic 1 "Denmark1" or "Denmark" "Denmark1"
+"""
+
+parser = zulip.add_default_arguments(argparse.ArgumentParser(usage=usage))
+parser.add_argument('stream_id')
+parser.add_argument('topic_name', type=str)
+options = parser.parse_args()
+
+client = zulip.init_from_options(options)
+if (type(options.stream_id) == str):
```

Comment:
@Pradhvan Have you tried manually testing this ? I am getting an error while trying to pass a stream id (i.e. int) as argument. Seems like it treats it as string.

Reply:
@akashaviator let me try manually testing it. 😄

Reply:
@Pradhvan have you tested this?  

If you don't have time, can @akashaviator take over finishing this?

Reply:
@timabbott sorry for the delay. I will test and update this. So that @akashaviator can review and merge this.

Reply:
@akashaviator can you take this over? Sorry but I am bit stuck on things and won't be able to give this time.

Reply:
Since it's not merged, I can take this up again. @akashaviator

Reply:
@Pradhvan  are you working on this? Just asking since i did not see it claimed. Thanks!